external_distributions/README.md

```diff
@@ -63,21 +63,23 @@ If you cannot obtain a recent enough version of CMake via your favourite package
-| `Boost` (**highly recommended**) | [1.83.0](https://github.com/boostorg/boost/tree/boost-1.83.0)                                                                             |
-| `CGAL` (**highly recommended**)  | [6.0.1](https://github.com/CGAL/cgal/tree/v6.0.1)                                                                                         |
-| `GKlib`                          | [commit `6e7951358fd896e2abed7887196b6871aac9f2f8`](https://github.com/KarypisLab/GKlib/tree/6e7951358fd896e2abed7887196b6871aac9f2f8)    |
-| `METIS`                          | [commit `a6e6a2cfa92f93a3ee2971ebc9ddfc3b0b581ab2`](https://github.com/KarypisLab/METIS/tree/a6e6a2cfa92f93a3ee2971ebc9ddfc3b0b581ab2)    |
-| `ParMETIS`                       | [commit `83bb3d4f5b2af826d0683329cad1accc8d829de2`](https://github.com/puneetmatharu/ParMETIS/tree/83bb3d4f5b2af826d0683329cad1accc8d829de2) |
-| `SuperLU`                        | [v6.0.1](https://github.com/xiaoyeli/superlu/tree/v6.0.1)                                                                                 |
-| `SuperLU_DIST`                   | [v9.1.0](https://github.com/xiaoyeli/superlu_dist/tree/v9.1.0)                                                                            |
-| `MUMPS`                          | [5.6.2](https://github.com/puneetmatharu/mumps/tree/v5.6.2.5)                                                                             |
-| `HYPRE`                          | [2.32.0](https://github.com/hypre-space/hypre/tree/v2.32.0)                                                                               |
-| `Trilinos`                       | [16.0.0](https://github.com/trilinos/Trilinos/tree/trilinos-release-16-0-0)                                                               |
+The table below lists the version of the third party libraries that work with(and/or are required by) `oomph-lib` and indicates which ones we can install for you via CMake. By default, we build "everything we can"; see below for how to customise the build process and select a serial or an MPI build. If you wish to provide your own version of these libraries, make sure it is the right version.
+
+
+| Library        | Required/optional | Built by default (serial build)? | Built by default (MPI build)?  | Version |
+| ----           | ---               | -----                    | ---                    | ---     | 
+| `OpenBLAS`     | required by `oomph-lib`              | Yes (though not on macOS; see below) | Yes (though not on macOS; see below) |  [0.3.25](https://github.com/OpenMathLib/OpenBLAS/tree/v0.3.29)      |
+| `SuperLU`       | required by `oomph-lib`               | Yes | Yes | [v6.0.1](https://github.com/xiaoyeli/superlu/tree/v6.0.1) | 
+| `METIS`        | required by `oomph-lib` (via `SuperLU`) | Yes | Yes | [commit `a6e6a2cfa92f93a3ee2971ebc9ddfc3b0b581ab2`](https://github.com/KarypisLab/METIS/tree/a6e6a2cfa92f93a3ee2971ebc9ddfc3b0b581ab2)  |              
```

Comment:
Admittedly these commit hashes are a little obtrusive... We can shorten these to the first 7 characters. Would you be able to update the rows that have commit hashes in them?